assets/src/edit-story/app/highlights/quickActions/useQuickActions.js

```diff
@@ -85,31 +187,75 @@ const useQuickActions = () => {
   const backgroundElement =
     currentPage?.elements.find((element) => element.isBackground) ||
     selectedElements?.[0]?.isBackground;
+  const selectedElement = selectedElements?.[0];
 
   const defaultActions = useMemo(
     () => [
       {
         Icon: Bucket,
         label: ACTION_TEXT.CHANGE_BACKGROUND_COLOR,
         onClick: handleFocusPageBackground(backgroundElement?.id),
+        onMouseDown: handleMouseDown,
       },
       {
         Icon: Media,
         label: ACTION_TEXT.INSERT_BACKGROUND_MEDIA,
         onClick: handleFocusMediaPanel(),
+        onMouseDown: handleMouseDown,
         separator: 'top',
       },
       {
         Icon: LetterTPlus,
         label: ACTION_TEXT.INSERT_TEXT,
         onClick: handleFocusTextSetsPanel(),
+        onMouseDown: handleMouseDown,
       },
     ],
     [
       backgroundElement?.id,
       handleFocusMediaPanel,
       handleFocusPageBackground,
       handleFocusTextSetsPanel,
+      handleMouseDown,
+    ]
+  );
+
+  const foregroundImageActions = useMemo(
+    () => [
+      {
+        Icon: PictureSwap,
+        label: ACTION_TEXT.REPLACE_MEDIA,
+        onClick: handleFocusMedia3pPanel(selectedElement?.id),
```

Comment:
👍  I'll swap this out after I have access to it in my branch for background image to focus on whatever panel the media came from.